x/nft/keeper/keeper_test.go

```diff
@@ -0,0 +1,363 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	tmtime "github.com/tendermint/tendermint/libs/time"
```

Comment:
We are using v0.34.14 of tendermint, however,`tendermint/libs/time` is only in [v0.35.1](https://pkg.go.dev/github.com/tendermint/tendermint?tab=versions) of tendermint

Reply:
should be solved with the protocgen.sh that uses third_party/proto